README.md

```diff
@@ -60,25 +53,25 @@ or in the `.yml` config.
 jtype: Flow
 pods:
   - name: encoder
-    uses: 'jinahub://MyDummyExecutor'
+    uses: 'jinahub://SpacyTextEncoder'
 ```
 
 
 ### 📦️ Via Pypi
 
-1. Install the `jinahub-MY-DUMMY-EXECUTOR` package.
+1. Install the `jinahub-spacy-text-encoder` package.
 
 	```bash
-	pip install git+https://github.com/jina-ai/EXECUTOR_REPO_NAME.git
+	pip install git+https://github.com/jina-ai/executor-text-spacy-encoder.git
 	```
 
-1. Use `jinahub-MY-DUMMY-EXECUTOR` in your code
+1. Use `jinahub-spacy-text-encoder` in your code
 
 	```python
 	from jina import Flow
-	from jinahub.SUB_PACKAGE_NAME.MODULE_NAME import MyDummyExecutor
+	from jinahub.encoder.spacy_text_encoder import SpacyTextEncoder
```

Comment:
I think this is not accurate, look at the `setup.py`

Reply:
removing the 'text' in the path following 2.0 executor guideline

Reply:
Then change it in setup and in the tests